gunicorn/arbiter.py

```diff
@@ -574,6 +574,7 @@ def manage_workers(self):
         workers = self.WORKERS.items()
         workers = sorted(workers, key=lambda w: w[1].age)
         while len(workers) > self.num_workers:
+            self.log("KILLING WORKER because there are {} when there should be {}", len(workers), self.num_workers)
```

Comment:
```suggestion
            self.log.info("KILLING WORKER because there are {} when there should be {}", len(workers), self.num_workers)
```

Reply:
I think these need to be `%d` or `%s` as well, the `{}` is for calls to `format()`

(though you could also just use an f-string and let that handle the formatting/interpolation)